docs/pt-br/manage/versions.md

```diff
@@ -86,7 +86,7 @@ asdf local <name> latest[:<version>]
 
 `local` escreve a versão para `$PWD/.tool-versions`, crie se necessário .
 
-Veja em `.tool-versions` [arquivo de seleção de configuração](/pt-br/core-configuration) para mais detalhes.
+Veja em `.tool-versions` [arquivo de seleção de configuração](/docs/pt-br/core-configuration) para mais detalhes.
```

Comment:
For all additions of the `/docs/` segment, same question as above. Even for other languages the `/docs/` segment isn't part of the canonical URL.